xocto/ranges.py

```diff
@@ -832,3 +832,9 @@ def get_finite_datetime_ranges_from_timestamps(
         # or end of the period.
         if start != end
     ]
+
+
+def period_is_fully_covered(period: Range[T], ranges: Sequence[Range[T]]) -> bool:
```

Comment:
🐼 I think the naming here isn't quite right (periods are related to temporal data).

How about `is_subrange`?

Reply:
This function was removed -> https://github.com/octoenergy/xocto/pull/69#discussion_r1186008306